upgrade_test.py

```diff
@@ -530,6 +530,10 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
         InfoEvent(message='pre-test - prepare test keyspaces and tables').publish()
         # prepare test keyspaces and tables before upgrade to avoid schema change during mixed cluster.
         self.prepare_keyspaces_and_tables()
+        InfoEvent(message='Running s-b to create schemas to avoid #11459').publish()
+        large_partition_stress_during_upgrade = self.params.get('stress_before_upgrade')
+        sb_create_schema = self.run_stress_thread(stress_cmd=large_partition_stress_during_upgrade, duration=1)
+        self.verify_stress_thread(sb_create_schema)
```

Comment:
this probably make test run longer. do we know how much? why not creating tables only without stress?

Reply:
`fill_and_verify_db_data` method will run when the new 1-minute load finishes.
So, the `fill_and_verify_db_data` is not affected by this new change.

Reply:
> test on https://jenkins.scylladb.com/job/scylla-staging/job/fabio/job/issue_11459_reproducers_and_staging/job/rolling-upgrade-ubuntu18.04-test/2/ passed with success (failed to clean resources and to send email. i will fix the commit message, so it can be merged

you can check yourself if it added too much or not, but the test took overall 5.3 hours.. what seems to be good for me

Reply:
According to the current SCT code, the duration will be calculated as "1*60 + 600" seconds. It is 11 minutes.
@fgelcer so, do we want to spend 11 minutes here?

Reply:
AFAICT this change is not going to be merged soon... as the issue is being chased, and i propose here this workaround, until the issue is fixed... but for now, as we are trying to reproduce it, this will stay as is